docs/reference/msbuild-targets.md

```diff
@@ -33,93 +34,94 @@ Similarly, you can write an MSBuild task, write your own target and consume NuGe
 > [!NOTE]
 > `$(OutputPath)` is relative and expects that you are running the command from the project root.
 
-## pack target
+## NuGet pack target
```

Comment:
Not sure NuGet adds much here.

Reply:
I was trying to optimize for SEO, by adding 'NuGet' to avoid confusion between a  MSbuild pack target or a NuGet pack target.

See point 4:
https://www.titangrowth.com/blog/capitalize-on-content-10-techniques-for-seo-content-writing/#:~:text=Discover%20SEO%20Writing%20Tips%20&%20Techniques.%201%201.,both%20your%20readers%20and%20the%20...%20More%20items

Reply:
There's no difference between MSBuild and NuGet pack target, it's only 1. 

Hoping some docs folks can comment on the header given that NuGet is everywhere through the file. 

Also (use incognito).

https://www.bing.com/search?q=nuget+pack+target&qs=n&form=QBRE&sp=-1&pq=nuget+pack+targe&sc=3-16&sk=&cvid=5E53D82631394A23BBA2548D5DF1EDDE

https://www.google.com/search?q=nuget+pack+target&rlz=1C1CHBF_enUS732US732&oq=nuget+pack+target&aqs=chrome..69i57j69i64j69i61l2j69i60.2279j0j7&sourceid=chrome&ie=UTF-8

The top hit is the same.

Reply:
I've been trying to follow advice on H2 titles:

https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-basics?branch=master#make-an-article-scannable-h2s

Reply:
Reverting title changes.

Optimizing for SEO requires analyze the whole page. It is worth to describe in another issue. https://github.com/NuGet/Client.Engineering/issues/794